clients/java/client/src/it/java/org/camunda/bpm/client/client/ClientIT.java

```diff
@@ -39,19 +43,13 @@
 
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Properties;
+import java.util.*;
```

Comment:
We try to avoid star imports, as they might create conflicts

Reply:
good point, its done :slightly_smiling_face: